cluster/gce/config-default.sh

```diff
@@ -255,6 +255,9 @@ if [[ (( "${KUBE_FEATURE_GATES:-}" == *"AllAlpha=true"* ) || ( "${KUBE_FEATURE_G
   RUN_CONTROLLERS="${RUN_CONTROLLERS:-*,endpointslice}"
 fi
 
+# By default disable gkenetworkparamset controller in CCM
+RUN_CCM_CONTROLLERS="${RUN_CCM_CONTROLLERS:-*,-gkenetworkparamset}"
```

Comment:
"-*" would disable all controllers?

Reply:
I think that the minus belong to the the bash `:-` for defaulting, but don't know if this can be interpreted differently

Reply:
Yes, it's for defaulting (and the same pattern as with `RUN_CONTROLLERS`).

No, `-*` will not disable controllers, CCM will throw error: ```
Error: "*" is not in the list of known controllers
```

Reply:
However, setting `--controllers=` to empty string would disable all controllers.